cfg/excalibur/jec17ul/data17_09Aug2019/_make_configs.py

```diff
@@ -17,17 +17,22 @@
 
 RUN='{run}'
 CH='{ch}'
-JEC='{{}}_Run{{}}_{{}}_{jecv_suffix}'.format(JEC_BASE, RUN, JEC_VERSION)
+#JEC='{{0}}_Run{{1}}_{{2}}_{jecv_suffix}'.format(JEC_BASE, RUN, JEC_VERSION)
+JEC='{{0}}_Run{{1}}_{{2}}'.format(JEC_BASE, RUN, JEC_VERSION)
```

Comment:
Do you really need double brackets?

Reply:
Yes since there are two format steps, one for the generation of the configs and another inside the configs.